code/crates/core-driver/src/driver.rs

```diff
@@ -197,11 +202,6 @@ where
         &self.validator_set
     }
 
-    /// Return recorded evidence of equivocation for this height.
```

Comment:
This method was never used and is now redundant as the proposal evidence can be gather from the proposal keeper now accessible via `.proposals()`.